tests/check_convert_util.c

```diff
@@ -31,10 +31,29 @@
 
 #include <check.h>
 #include <stdlib.h>
+#include "convert_util.h"
 
 START_TEST(test_convert_parse_header){
-	/* TODO */
-	ck_assert(1);
+	int			ret;
+	struct convert_header	hdr;
+	uint8_t *		buff = (uint8_t *)&hdr;
+	size_t			tlvs_length;
+
+	ret = convert_parse_header(buff, sizeof(hdr) - 1, &tlvs_length);
+	ck_assert_msg(ret == -1, "Should fail: buff too short");
+
+	ret = convert_parse_header(buff, sizeof(hdr) + 1, &tlvs_length);
+	ck_assert_msg(ret == -1, "Should fail: buff too long");
+
+	hdr.version	= CONVERT_VERSION + 1;
+	ret		= convert_parse_header(buff, sizeof(hdr), &tlvs_length);
+	ck_assert_msg(ret == -1, "Should fail: unsupported version");
+
+	hdr.version		= CONVERT_VERSION;
+	hdr.total_length	= 2;
+	ret			= convert_parse_header(buff, sizeof(hdr),
+	                                               &tlvs_length);
+	ck_assert(ret == 0 && tlvs_length == 4);
```

Comment:
can you just comment why `tlvs_length == 4` (and use `ck_assert_msg` with an explanation why it should be that)

Reply:
Sure! Will do.